src/content/LandingPage/LandingPage.js

```diff
@@ -1,7 +1,103 @@
 import React from 'react';
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  Button,
+  Tabs,
+  Tab,
+} from 'carbon-components-react';
 
 const LandingPage = () => {
-  return <div>LANDING PAGE</div>;
+  const props = {
+    tabs: {
+      selected: 0,
+      triggerHref: '#',
+      role: 'navigation',
+    },
+    tab: {
+      href: '#',
+      role: 'presentation',
+      tabIndex: 0,
+    },
+  };
+  return (
+    <div className="bx--grid bx--grid--full-width landing-page">
+      <div className="bx--row landing-page__banner">
+        <div className="bx--row landing-page__banner">
```

Comment:
Your duplicate `<div>` wrappers with the class `landing-page__banner` is causing extra padding in this row.